dockerfiles/QAT/README.md

```diff
@@ -0,0 +1,59 @@
+
+  * [Introduction](#introduction)
+  * [Building](#building)
+  * [Options](#options)
+
+## Introduction
+
+The files in this directory can be used to build a modified Kata Containers rootfs
+and kernel with modifications to support Intel QAT hardware.
+
+The generated files will need to be copied and configured into your Kata Containers
+setup.
+
+Please see the
+[Kata QAT documentation](https://github.com/kata-containers/documentation/blob/master/use-cases/using-Intel-QAT-and-kata.md)
+for more specific QAT details.
+
+## Building
+
+The process is executed using Docker:
+
+```sh
+$ cd ..
```

Comment:
Where are we? :smile: Can you `cd` to a known directory as it's unclear what directory the user is/should be in at this point.

Reply:
Well this is highly unusual I think. Maybe if I turn the question round: why do we need to change directory? I thought idocker would build an image if it found `Dockerfile` in the current directory, so why leave it?